### DISCUSSION TITLE
q-dir 11.14 hash check failed

### DISCUSSION THREAD

**Discussion by ooooo84** [ScoopInstaller/Extras#9757] [General]
Q-dir is updated to 11.14 now!

ERROR Hash check failed!
App:         extras/q-dir
URL:         https://www.softwareok.com/Download/Q-Dir_Portable_x64.zip
First bytes: 50 4B 03 04 0A 00 00 00
Expected:    a4ad2a9d088eb907bcbe54257db605a22a6a072ef4943b281e782c510174d315
Actual:      ebbfbcc77e78019d667c1658e0752bd1073d87e2dad2f8f1bc3aff94b93e16a6

**Comment by niheaven** [comment]
Please don't submit issues in Discussion.
